cdap-ui/app/features/workflows/workflow.less

```diff
@@ -12,4 +13,58 @@ body.state-workflows{
       }
     }
   }
+
+
+  .schedule-heading {
+    background-color: white;
+    border: 1px solid @border-color;
+    border-top-left-radius: 5px;
```

Comment:
Can we use ```@border-radius-base``` here? Think we use the one from bootstrap (if not we need to change that) We should probably have border-radius unified across everywhere (except top nav)